clang/test/CIR/CodeGen/X86/avx512f-builtins.c

```diff
@@ -77,3 +77,27 @@ __m512i test_mm512_undefined_epi32(void) {
   // OGCG: ret <8 x i64> zeroinitializer
   return _mm512_undefined_epi32();
 }
+
+__mmask16 test_mm512_kunpackb(__mmask16 A, __mmask16 B) {
+  // CIR-LABEL: test_mm512_kunpackb
```

Comment:
For CIR you need to check what's generated for the wrapper function. In this case, `_mm512_kunpackb`